### PR TITLE
Create probation tiering dev namespace

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+variable "application" {
```
Comment (poornima-krishnasamy) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf:1]: @nickmcmahon01 , can you please update the variables to have snake case. We are trying to keep the consistency. You can refer the CLI template file in here: https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespace-resources-cli-template/resources/variables.tf

Comment (nickmcmahon01) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf:1]: no problem, I copied an existing one so i'll go through and do that now.

**Review by digitalronin — CHANGES_REQUESTED** [review]
There are some errors in the plan pipeline [here](https://concourse.cloud-platform.service.justice.gov.uk/builds/1358)
```
Command: kubectl -n probation-tiering-dev apply --dry-run -f namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev failed.
the namespace from the provided object "probation_tiering_dev" does not match the namespace "probation-tiering-dev". You must pass '--namespace=probation_tiering_dev' to perform this operation.
```
I think you changed the namespace name by mistake, when you renamed your variables to snake case.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  default = "probation-tiering"
+}
+
+variable "namespace" {
+  default = "probation_tiering_dev"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-tiering-dev/resources/variables.tf:6]: this needs to be `probation-tiering-dev` - it's the namespace name, and that's hyphenated.

**Review by digitalronin — CHANGES_REQUESTED** [review]

**Review by digitalronin — APPROVED** [review]

**Pull request merged**